pkg/ocm/list.go

```diff
@@ -0,0 +1,41 @@
+package ocm
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// ListPoliciesInAllNamespaces returns a cluster-wide policy inventory.
+func ListPoliciesInAllNamespaces(apiClient *clients.Settings, options metaV1.ListOptions) ([]*PolicyBuilder, error) {
```

Comment:
It seems that you are not using the options parameter that is provided

Reply:
Correct, thanks.